### ISSUE TITLE
Property 'children' will overwrite the base property in 'Pattern'

### ISSUE THREAD

**Issue by ealib** [Eyal-Shalev/docopt.js#9]
With new Deno 1.9.0, a TypeScript error is found in [abstract class `ChildPattern`](https://deno.land/x/docopt@v1.0.6/src/pattern.ts#L111):

```
error: TS2612 [ERROR]: Property 'children' will overwrite the base property in 'Pattern'. If this is intentional, add an initializer. Otherwise, add a 'declare' modifier or remove the redundant declaration.
  public children: undefined;
         ~~~~~~~~
    at https://deno.land/x/docopt@v1.0.6/src/pattern.ts:111:10
```

**Comment by Eyal-Shalev** [comment]
Thanks, I'll look into this

**Comment by shah** [comment]
Thanks for checking, @Eyal-Shalev I'm getting the same error in my code with Deno 1.9.0.

**Comment by Eyal-Shalev** [comment]
@masnagam opened a PR for this that seems to fix the issue #10 
I'll try to release a new version in the next couple of days.

**Comment by Eyal-Shalev** [comment]
Fixed in 1.0.7
